sharding-core/sharding-core-preprocessor/src/test/java/org/apache/shardingsphere/core/optimize/segment/table/TablesContextTest.java

```diff
@@ -133,6 +138,50 @@ public void assertNotFoundTable() {
         assertFalse(table.isPresent());
     }
     
+    @Test
+    public void assertFindTableNameWhenSingleTable() {
+        SelectStatement selectStatement = new SelectStatement();
+        selectStatement.getAllSQLSegments().add(createTableSegment("table_1", "tbl_1"));
+        assertTrue(new TablesContext(selectStatement).findTableName(null, null).isPresent());
+    }
+    
+    @Test
+    public void assertFindTableNameWhenColumnSegmentOwnerPresent() {
+        SelectStatement selectStatement = new SelectStatement();
+        selectStatement.getAllSQLSegments().add(createTableSegment("table_1", "tbl_1"));
+        selectStatement.getAllSQLSegments().add(createTableSegment("table_2", "tbl_2"));
+        TablesContext tablesContext = new TablesContext(selectStatement);
+        ColumnSegment columnSegment = mock(ColumnSegment.class);
+        when(columnSegment.getOwner()).thenReturn(Optional.of(new TableSegment(0, 10, "table_1")));
+        assertTrue(tablesContext.findTableName(columnSegment, null).isPresent());
+    }
+    
+    @Test
+    public void assertFindTableNameWhenColumnSegmentOwnerAbsent() {
+        SelectStatement selectStatement = new SelectStatement();
+        selectStatement.getAllSQLSegments().add(createTableSegment("table_1", "tbl_1"));
+        selectStatement.getAllSQLSegments().add(createTableSegment("table_2", "tbl_2"));
+        TablesContext tablesContext = new TablesContext(selectStatement);
+        ColumnSegment columnSegment = mock(ColumnSegment.class);
+        when(columnSegment.getOwner()).thenReturn(Optional.<TableSegment>absent());
+        TableMetas tableMetas = mock(TableMetas.class);
+        when(tableMetas.containsColumn(anyString(), anyString())).thenReturn(false);
```

Comment:
Can you have a try for remove this line? mock result with false is default action, maybe we do not need to mock it.